resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -869,7 +868,7 @@ private[spark] class ApplicationMaster(
           }
         } else {
           logError(log"Application Master lost connection with driver! Shutting down. " +
-            log"${MDC(LogKeys.REMOTE_ADDRESS, remoteAddress)}")
+            log"${MDC(LogKeys.HOST_PORT, remoteAddress)}")
```

Comment:
why renaming this one?

Reply:
I remember there was a PR before that had already unified `REMOTE_ADDRESS` as `HOST_PORT`